environment.yml

```diff
@@ -1,13 +1,10 @@
-name: nb-skdecide092-py38
+name: nb-skdecide-py38
 dependencies:
   - python=3.8
   - pip
   - pip:
-    - matplotlib
     - ipywidgets
     - ipympl
-    - scikit-decide[all]==0.9.2
-    - nbgitpuller
```

Comment:
We do not want to remove nbgitpuller. This is the package used to pull the notebooks from appropriate branch when launching jupyter on binder. (we do neither want to remove matlplotlib as it is necessary and we want to keep as many dependencies in the environment as possible to avoid installing too much at starting time)